src/plugins/orangehrmLeavePlugin/Service/Model/LeaveListModal.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+
+namespace OrangeHRM\Leave\Service\Model;
+
+use OrangeHRM\Core\Api\V2\Serializer\ModelTrait;
+use OrangeHRM\Core\Api\V2\Serializer\Normalizable;
+use OrangeHRM\Core\Traits\Auth\AuthUserTrait;
+use OrangeHRM\Core\Traits\Service\ConfigServiceTrait;
+use OrangeHRM\Entity\Leave;
+
+class LeaveListModal implements Normalizable
+{
+    use ModelTrait;
+    use ConfigServiceTrait;
+    use AuthUserTrait;
+
+    public function __construct(Leave $leave)
+    {
+        $this->setEntity($leave);
+        $filter = [
+            'id',
+            ['getDecorator', 'getLeaveDate'],
+            'lengthHours',
+            ['getEmployee','getEmpNumber'],
+            ['getEmployee','getLastName'],
+            ['getEmployee','getFirstName'],
+            ['getEmployee','getMiddleName'],
+            ['getEmployee','getEmployeeId'],
```

Comment:
one field missing. termination id. this used to show whether the employee is past employee.
if this param null means active employee. if this param has value then we are showing `(Past Employee)` suffix